view/rooms/app.js

```diff
@@ -485,6 +489,68 @@ function createIframeEvent(channel, payload, bots, botId) {
   return 'Success';
 }
 
+
+/**
+ * The user has entered the page log this event
+ *
+ * @returns {Promise} For use in chaining.
+ */
+function userEnterRoom() {
+  const currentUser = {
+    name: _user.name,
+    id: _user.id,
+    email: _user.email,
+    fullName: _user.fullName,
+  }
+
+  const message = currentUser.fullName + ' has joined the room at ' +
+      moment().format('YYYY-MM-DD HH:mm Z');
+  const eventType =  {
+      'type': 'User',
+      'user': currentUser,
+      'isActive': true,
+    }
```

Comment:
style

Reply:
@BrandonMurnane Some users who did not sign in with SSO may not have a fullName field, we should probably fall back to name/email for this.